### ISSUE TITLE
`.xmp` files which are not named identically to an image file will be silently ignored

### ISSUE THREAD

**Issue by qlyoung** [ivandokov/phockup#211]
Due to the way `.xmp` files are handled, any `.xmp` files which do not have a corresponding file named with an identical filename (excluding the `.xmp` extension) will be skipped.

This is contrary to the behavior described in the `README` which states:

> All files which are not images or videos or those which do not have creation date information will be placed in a directory called unknown without file name change. By doing this you can be sure that the input directory can be safely deleted after the successful process completion because all files from the input directory have a copy in the output directory.

Example:
```
qlyoung@host /m/qldata> mkdir -p test/in test/out; cd test
qlyoung@host /m/q/test> touch in/hi.jpg in/hello.xmp
qlyoung@host /m/q/test> docker run -v .:/mnt phockup /mnt/in /mnt/out
[2023-09-01 22:12:25] - [INFO] - /mnt/in/hi.jpg => /mnt/out/unknown/hi.jpg
[2023-09-01 22:12:25] - [INFO] - Processed 1 files in 0.11 seconds. Average Throughput: 8.99 files/second
[2023-09-01 22:12:25] - [INFO] - Copied 1 files.
qlyoung@host /m/q/test> tree out
out
└── unknown
    └── hi.jpg

1 directory, 1 file
qlyoung@host /m/q/test> tree in
in
├── hello.xmp
└── hi.jpg

0 directories, 2 files
```

Test conducted on cdbc8d43fafa54192687ce6db8509a15b85c4cfc

**Comment by ivandokov** [comment]
You are right. XMP files are an exception to that statement as they are meta files alongside the images. The readme should be changed to state that.

**Comment by ivandokov** [comment]
Please add the change to PR #210 as this new feature will alter the text that has to be added to the readme to include more sidecar extentions.
